README.md

```diff
@@ -22,7 +22,24 @@ The full reference documentation for rules is at https://haskell.build.
 
 ## Setup
 
-You'll need [Bazel >= 0.27][bazel-getting-started] installed.
+You'll need [Bazel >= 0.29][bazel-getting-started] installed.
+
+If you are on NixOS, skip to the [Nixpkgs](#Nixpkgs) section.
```

Comment:
The exact same line below the windows section should probably be removed.

Reply:
Good catch. Removed.